src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -26,7 +26,58 @@ def setUp(self):
 
     def tearDown(self):
         pass
-		
+    
+    def assertCbsdRecord(self, registration_request, grant_request, grant_response, recordData):
+        for index, device in enumerate(registration_request):
+            record_id = 'cbsd/'+ device['fccId']+'/'+ hashlib.sha1(device['cbsdSerialNumber']).hexdigest()
+            cbsd_record = [record['registration'] for record in recordData if record['id'] == record_id]
+            self.assertEqual(1, len(cbsd_record))
+            self.assertEqual(device['cbsdCategory'], cbsd_record[0]['cbsdCategory'])
+            self.assertEqual(device['fccId'], cbsd_record[0]['fccId'])
+            self.assertEqual(device['callSign'], cbsd_record[0]['callSign'])
+            self.assertEqual(device['airInterface']['radioTechnology'],\
+                             cbsd_record[0]['airInterface']['radioTechnology'])
+            self.assertEqual(device['installationParam']['latitude'],\
+                             cbsd_record[0]['installationParam']['latitude'])
+            self.assertEqual(device['installationParam']['longitude'],\
+                             cbsd_record[0]['installationParam']['longitude'])
+            self.assertEqual(device['installationParam']['height'],\
+                             cbsd_record[0]['installationParam']['height'])
+            self.assertEqual(device['installationParam']['heightType'], \
+                             cbsd_record[0]['installationParam']['heightType'])
```

Comment:
Please review the SAS-SAS spec and check that you are included all required parameters. I think there are at least five missing.

Reply:
I think I added all the necessary parameters, for the grouping parameters the CBSDs that we added don't have any grouping param, so I think we don't need to verify this, right?

Reply:
It would be better to write this function more generally since we will need to extend the test case to be configurable. It's OK to check in a static version for now but once that's done we'll want to modify it to be configurable since that's what's required by the test spec. Hopefully that makes my other pedantic comments make a little more sense :)